test/testing_suite_test.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:test/test.dart';
+
+import 'utils/client.dart'; // testServer
+import "tests/analytics.dart"; // versionTests(), analyticsTests()
+import "tests/documents.dart"; // documentsTests()
+import "tests/dump.dart"; // dumpTests()
+import "tests/exceptions.dart"; // exceptionsTests()
+import "tests/get_client_stats.dart"; // getClientStatsTests()
+import "tests/get_keys.dart"; // getKeysTests()
+import "tests/get_version.dart"; //getVersionTests()
+import "tests/health.dart"; // healthTests(), healthFailTests()
+import "tests/indexes.dart"; // indexesTests()
+import "tests/search.dart"; // searchTests()
+import "tests/settings.dart"; // settingsTests()
+
+void main() {
```

Comment:
My only concern regarding this change is related to the next tests we will add in the future...
We will need to add them here, right?